graphqlapi/src/main/java/io/stargate/graphql/PlaygroundServlet.java

```diff
@@ -16,562 +16,43 @@
-                  + "          style=\"transform: translate(100px, 100px);\"></path>\n"
-                  + "        <path class=\"iEGVWn\" stroke=\"#fff\" stroke-width=\"4\" stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M63.55 27.5l32.9 19-32.9-19z\"></path>\n"
-                  + "        <path class=\"bsocdx\" stroke=\"#fff\" stroke-width=\"4\" stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M96 46v38-38z\"></path>\n"
-                  + "        <path class=\"jAZXmP\" stroke=\"#fff\" stroke-width=\"4\" stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M96.45 84.5l-32.9 19 32.9-19z\"></path>\n"
-                  + "        <path class=\"hSeArx\" stroke=\"#fff\" stroke-width=\"4\" stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M64.45 103.5l-32.9-19 32.9 19z\"></path>\n"
-                  + "        <path class=\"bVgqGk\" stroke=\"#fff\" stroke-width=\"4\" stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M32 84V46v38z\"></path>\n"
-                  + "        <path class=\"hEFqBt\" stroke=\"#fff\" stroke-width=\"4\" stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M31.55 46.5l32.9-19-32.9 19z\"></path>\n"
-                  + "        <path class=\"dzEKCM\" id=\"Triangle-Bottom\" stroke=\"#fff\" stroke-width=\"4\" d=\"M30 84h70\" stroke-linecap=\"round\"></path>\n"
-                  + "        <path class=\"DYnPx\" id=\"Triangle-Left\" stroke=\"#fff\" stroke-width=\"4\" d=\"M65 26L30 87\" stroke-linecap=\"round\"></path>\n"
-                  + "        <path class=\"hjPEAQ\" id=\"Triangle-Right\" stroke=\"#fff\" stroke-width=\"4\" d=\"M98 87L63 26\" stroke-linecap=\"round\"></path>\n"
-                  + "      </g>\n"
-                  + "    </svg>\n"
-                  + "    <div class=\"text\">Loading\n"
-                  + "      <span class=\"dGfHfc\">GraphQL Playground</span>\n"
-                  + "    </div>\n"
-                  + "  </div>\n"
-                  + "\n"
-                  + "  <div id=\"root\" />\n"
-                  + "  <script type=\"text/javascript\">\n"
-                  + "    window.addEventListener('load', function (event) {\n"
-                  + "\n"
-                  + "      const loadingWrapper = document.getElementById('loading-wrapper');\n"
-                  + "      loadingWrapper.classList.add('fadeOut');\n"
-                  + "\n"
-                  + "\n"
-                  + "      const root = document.getElementById('root');\n"
-                  + "      root.classList.add('playgroundIn');\n"
-                  + "\n"
-                  + "      GraphQLPlayground.init(root, {\n"
-                  + "        // you can add more options here\n"
-                  + "      })\n"
-                  + "    })\n"
-                  + "  </script>\n"
-                  + "</body>\n"
-                  + "</html>");
+    try (InputStream is = entry.openConnection().getInputStream()) {
```

Comment:
Good idea to use a resource file.

We might need some templating functionality to replace values server side (i.e. keyspace name) but its definitely an improvement.